velox/exec/fuzzer/MemoryArbitrationFuzzer.cpp

```diff
@@ -326,6 +350,12 @@ std::vector<RowVectorPtr> MemoryArbitrationFuzzer::generateAggregateInput(
   return generateProbeInput(keyNames, keyTypes);
 }
 
+std::vector<RowVectorPtr> MemoryArbitrationFuzzer::generateRowNumberInput(
+    const std::vector<std::string>& keyNames,
+    const std::vector<TypePtr>& keyTypes) {
+  return generateProbeInput(keyNames, keyTypes);
```

Comment:
Can we have generateInput and generateProbeInput/generateAggregateInput/generateRowNumberInput all call into that function? Thanks!

Reply:
updated, please take another look, thanks!